README.md

```diff
@@ -28,7 +28,20 @@ git clone git@github.com:binpash/pash.git
 cd pash/scripts
 docker build -t "pash/18.04" .
 # Then launch container:
-docker run --name pash-play -it pash/18.04
+```
+
+Then, depending on the OS, run the dockercontainer. Note to change `ubuntu` to `fedora` or `debian` as required.
+
+```sh
+docker run -f ./docker/ubuntu/Dockerfile --name pash-play -it pash:latest
```

Comment:
The `-f ` is not supported by `docker run` (at least in my setup). It is only supported by build. Did you intend to add it there?

Reply:
Committed again to this branch. Should be resolved now.